content/docs/reference/state.md

```diff
@@ -1,140 +1,307 @@
+- Secure checkpoint storage, with client-side authentication to your cloud provider
+- Encrypted state in transit and at rest
+- Full deployment history for auditing and rollback purposes
+- Concurrent state locking to prevent corrupting your infrastructure state in a team environment
+- Advanced policy and RBAC (Role Based Access Control)
 
-Although backends are primarily responsible for storing state reliably, the web backend also supports additional
-features that the filesystem backend does not. In particular, the web backend supports keeping track of full
-deployment history (for auditing and rollback purposes), and supports concurrent locking so that you don't accidentally
-corrupt your infrastructure state when using Pulumi in a team environment. It also supports advanced policy and RBAC.
+#### app.pulumi.com architecture
 
-The app.pulumi.com architecture can be visualized as follows:
+<img src="/images/docs/reference/state_saas.png" alt="Pulumi SaaS Architecture"
+class="img-bordered">
 
-<img src="/images/docs/reference/state_saas.png" alt="Pulumi SaaS Architecture" class="img-bordered">
+#### Enterprise web architecture
 
-The Pulumi Enterprise product offers self-hosting options for the web backend, if you wish to use these features
-without depending on app.pulumi.com. The Enterprise web architecture looks like the following:
+Pulumi enterprise users have a self-hosting option, for using the Pulumi backend features without depending on
+`app.pulumi.com`.
 
-<img src="/images/docs/reference/state_enterprise.png" alt="Pulumi Enterprise Architecture" class="img-bordered">
+<img src="/images/docs/reference/state_enterprise.png" alt="Pulumi Enterprise
+Architecture" class="img-bordered">
 
-For more information on Pulumi Enterprise, please [contact us](https://www.pulumi.com/pricing/#contact).
+[Contact us](https://www.pulumi.com/pricing/#contact) for more information on
+Pulumi Enterprise.
 
 ## Logging In
 
-The `pulumi login` command lets you log into a backend. By default, anytime you try to do something that requires
-stacks or state, you will be prompted to log in.
+The `pulumi login` <a href="{{< relref "/docs/reference/cli/pulumi_login" >}}"
+target="_blank">CLI command</a> lets you log in to a backend. By default,
+you will be prompted to log in anytime you try to do something that requires stacks or state.
 
-### To the Web Backend
+### Login and version check
 
-The web backend login process involves using access tokens. The prompt looks like the following:
+To quickly check your backend login info and the Pulumi version running on your machine, run the following:
+
+```bash
+$ pulumi whoami -v
+```
+
+See [pulumi whoami](/docs/reference/cli/pulumi_whoami) for more details.
+
+#### Example response
+
+```
+warning: A new version of Pulumi is available. To upgrade from version '0.17.22' to '0.17.25', run $ curl -sSL https://get.pulumi.com | sh
+or visit https://pulumi.io/install for manual instructions and release notes.
```

Comment:
remove the warning bit.  that's only there because your local pulumi is out of date.  we don't want that in the docs.